pkg/cmd/server/plugin/plugin.go

```diff
@@ -50,6 +50,10 @@ func NewCommand(f client.Factory) *cobra.Command {
 			f.SetClientQPS(config.ClientQPS)
 			f.SetClientBurst(config.ClientBurst)
 			pluginServer = pluginServer.
+				RegisterRestoreItemAction(
+					constant.PluginCSIPVCVSDURestoreRIA,
+					newPvcVSDVRestoreItemAction(f),
+				).
```

Comment:
I would move this a few lines down to just before the call to `newPvcRestoreItemAction(f)` at line 161 since they're related plugin actions.